Bio/Nexus/Nexus.py

```diff
@@ -100,12 +100,12 @@ def next_nonwhitespace(self):
         return None
 
     def skip_whitespace(self):
-        """Skips whitespace characters in NEXUS file."""
+        """Skip whitespace characters in NEXUS file."""
         while self.buffer[0] in WHITESPACE:
             self.buffer = self.buffer[1:]
 
     def next_until(self, target):
-        """Keeps iterating the NEXUS file until it reaches a target character.
+        """Keep iterating the NEXUS file until it reaches a target character.
```

Comment:
This wording was odd before your change, but how about "Iterate over the NEXUS file until a target character is reached."